src/borg/algorithms/msgpack/_version.py

```diff
@@ -2,5 +2,7 @@
 # Changes:
 # +borg1: drop support for old buffer protocol to be compatible with py310
 #         (backport of commit 9ae43709e42092c7f6a4e990d696d9005fa1623d)
-version = (0, 5, 6, '+borg1')
+# +borg2: fix runtime on big-endian OpenBSD/sparc64
+#         (revert of commit 9c514f0124abf03eef79e474299e2e3cf6381ea1)
```

Comment:
as we do not include master here, but 0.5.6, it would be better to point to the changes relative to 0.5.6.
so just give the changesets applied and skip the reverted one.

Reply:
Sure, will do.